README.md

```diff
@@ -51,6 +51,31 @@ fastify.register(fastifyStatic, {
 
 ```
 
+### Sending a file with `content-disposition` header
```

Comment:
Why would somebody do this over the simple `sendFile`?

Reply:
Following the issue, seeing the docs from Express, and taking a look into the purpose of the [`content-disposition`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Disposition) header, it seems that this is mostly to send files that should be threatened as an attachment by the browser itself for the final usage purposes.

This is the original purpose of the Express implementation so it might have the sense to also bring this out-of-the-box as a way of a plugin (my impression from the issue).